app/services/token_service.rb

```diff
@@ -3,12 +3,13 @@ class TokenService
   RANDOM_BYTES = 8
 
   class << self
-    def box(data)
+    def box(data, encryptor = nil)
+      encryptor = message_encryptor if encryptor.nil?
```

Comment:
This could be moved into the function arguments, but `message_encryptor` is a potentially very expensive call with a lot going on, so I opted to keep it in the function body.